### ISSUE TITLE
FFMPEG import error

### ISSUE THREAD

**Issue by SweetmanTech** [firebase/functions-samples#541]
### How to reproduce these conditions

1) clone git repo for `firebase/function-samples`
2) cd ffmpeg project
3) firebase add 
4) firebase deploy
5) ERROR

![image](https://user-images.githubusercontent.com/23249402/52598699-9d0bb000-2e24-11e9-94f5-dcbfed9ac0d0.png)

**Sample name or URL where you found the bug**
https://github.com/firebase/functions-samples/blob/Node-8/ffmpeg-convert-audio/functions/index.js

**Failing Function code used (including require/import commands at the top)**
```
const functions = require('firebase-functions');
const gcs = require('@google-cloud/storage')();
const path = require('path');
const os = require('os');
const fs = require('fs');
const ffmpeg = require('fluent-ffmpeg');
const ffmpeg_static = require('ffmpeg-static');
```

**Steps to set up and reproduce**

1) clone git repo for `firebase/function-samples`
2) cd ffmpeg project
3) firebase add 
4) firebase deploy
5) ERROR

**Sample data pasted or attached as JSON (not an image)**

**Security rules used**

### Debug output

**Errors in the [console logs](https://console.firebase.google.com/project/_/functions/logs?search=&severity=DEBUG)**

**Screenshots**

### Expected behavior

Deploy successfully to Firebase Cloud Functions!

### Actual behavior

ERROR - module import fails for firebase cloud storage

**Comment by shlomi-lachmish** [comment]
You can check issue #472

TLDR - change lines 19 & 57 with: 
```
const gcs = require('@google-cloud/storage');
const bucket = new gcs.Storage().bucket(fileBucket);
```
